vercel/turbo
# chore: add error to prefixed UI

### Description

In Go the prefix UI has output/warn/error methods that we use to write and prefix writes. This PR changes the prefixed UI in two external ways:
 - adds the error method
 - no longer require all prefixes to be provided, matching the behavior in Go

I'd suggest reviewing each commit on it's own as I tried to make sure each one was a contained change.

### Testing Instructions

Added new unit tests. The real testing will come when we use this in run and can check for Go parity.


Closes TURBO-1316
